docs/ui/schema-conv/spanner-draft.md

```diff
@@ -28,7 +28,7 @@ Users can view detailed information for a table by selecting it from the **Spann
 
 Column tab provides information on the columns that are a part of the selected table. It also provides the option to edit the column wherein a user can modify a column name, delete a column, change the data type of the column, add auto-generation to the column or modify the null property of the column. Once the user is done with required modifications, they can click on **SAVE & CONVERT **and the update would reflect in the session file and across all the components in the database.
 
-![](https://services.google.com/fh/files/misc/column-info-edit.png)
+![](https://services.google.com/fh/files/misc/dv1.png)
```

Comment:
can you please add an attachment in the PR description of how this page looks in the new png?

Reply:
Added